### PR TITLE
Gal 202/invitation clinic summary

### PR DESCRIPTION
**Pull request by msandhuNHS**
## Description

Added Clinic Summary page.

## Type of changes

- [ ] Refactoring (non-breaking change)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

**Review by joad432 — DISMISSED** [review]
Nice thank you.

File: package.json [hunk 1]
```diff
@@ -2,20 +2,31 @@
...
-    "lint": "next lint"
+    "lint": "next lint",
+    "test": "jest --watch"
   },
   "dependencies": {
-    "dotenv": "^16.3.1",
-    "next": "13.4.12",
+    "next": "13.4.9",
+    "nhsuk-frontend": "^7.0.0",
     "react": "18.2.0",
```
Comment (LaithM96) [inline package.json:16]: Decide on a versioning strategy. If you want to pin versions, do it for all packages. If you want to use semantic versioning, use it consistently.

File: package-lock.json [hunk 2]
```diff
@@ -1,427 +0,0 @@
-{
```
Comment (LaithM96) [inline package-lock.json:1]: why was this file deleted ?

File: gulpfile.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import gulp from 'gulp';
+import gulpSass from "gulp-sass";
+import * as sassCompiler from 'sass';
+
+const sass = gulpSass(sassCompiler);
+
```
Comment (LaithM96) [inline gulpfile.js:6]: Sass Compilation - Make sure to document how and when to run this task, especially if it's not integrated into your build process. Possible run book ?

File: .gitignore [hunk 4]
```diff
@@ -37,3 +37,6 @@ yarn-error.log*
 # typescript
 *.tsbuildinfo
 next-env.d.ts
+
```
Comment (LaithM96) [inline .gitignore:40]: Good practice to ignore these directories. Continue doing this.

File: next.config.js [hunk 5]
```diff
@@ -1,3 +1,5 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = { output: "export" };
-module.exports = nextConfig;
+
+const nextConfig = {};
```
Comment (LaithM96) [inline next.config.js:3]: Webpack Config: Ensure that the custom Webpack configuration is well-documented.

File: .editorconfig [hunk 6]
```diff
@@ -17,5 +17,5 @@ indent_size = 4
 [{Dockerfile,Dockerfile.}*]
 indent_size = 4
 
-[{Makefile,*.mk,go.mod,go.sum,*.go,.gitmodules}]
+[{Makefile,*.mk,go.mod,go.sum,*.go,.gitmodules,*.scss,*.css}]
```
Comment (LaithM96) [inline .editorconfig:20]: - Non-root User: Good practice, continue doing this.
- Environment Variables: Make sure these are documented for anyone using the Dockerfile.

File: package.json [hunk 7]
```diff
@@ -2,20 +2,31 @@
...
     "react-dom": "18.2.0"
   },
   "cacheDirectories": [
     "node_modules/",
```
Comment (LaithM96) [inline package.json:20]: The "cacheDirectories" key is not standard for a package.json file. ??

**Review by LaithM96 — COMMENTED** [review]
Can you please review my comments and then I will approve once I am satisfied with the comments :)

**Review by LaithM96 — APPROVED** [review]
happy to approve, well done, after the demo for this front end work.

**Review by LaithM96 — CHANGES_REQUESTED** [review]
dev 3 needs to be merged into main first.

File: .gitignore [hunk 8]
```diff
@@ -37,3 +37,6 @@ yarn-error.log*
...
+# package-lock
+/package-lock.json
```
[the remark this answers is not in the document]
Comment (zainm97) [reply .gitignore]: done

**Review by zainm97 — APPROVED** [review]

**Review by LaithM96 — APPROVED** [review]
approved, all good :)

**Pull request merged**